puppet/modules/slave/manifests/init.pp

```diff
@@ -55,6 +55,19 @@
     ensure  => absent,
   }
 
+  file { "${homedir}/.bundle":
+    ensure  => directory,
+    owner   => root,
```

Comment:
Making this owned by root means bundler can't create any files, like `~/.bundle/cache` and `~/.bundle/ruby`. That will likely cause problems.